charts/grafana/values.yaml

```diff
@@ -2,6 +2,11 @@ global:
   # -- Overrides the Docker registry globally for all images
   imageRegistry: null
 
+  # -- Overrides the Docker registries globally for all images based on their origin registry
+  imageRegistries: {}
```

Comment:
Please configure this in one of the ci test values.

You can use kiwigrid/k8s-sidecar as test, since it is published on multiple registries.